libs/ui/src/components/molecules/CheckboxWrapper/WrappedCheckbox.tsx

```diff
@@ -0,0 +1,42 @@
+import { useForm, Controller } from 'react-hook-form';
+import type { CheckboxWrapperProps } from '../../../types/formAndField';
+import { Checkbox } from '../../atoms/Checkbox';
+import { FieldWrapper } from '../inputWrapper/fieldWrapper';
+
+export const WrappedCheckbox = (props: CheckboxWrapperProps) => {
```

Comment:
How does this handle a single checkbox? Would that need to be passed in as an array?